src/app/shared/dialogs/dialogs-resources-viewer.component.ts

```diff
@@ -1,11 +1,29 @@
+import { Router, ActivatedRoute } from '@angular/router';
 import { Component, Inject } from '@angular/core';
-import { MAT_DIALOG_DATA } from '@angular/material';
+import { MAT_DIALOG_DATA, MatDialogRef } from '@angular/material';
+
 
 @Component({
-  template: `<planet-resources-viewer [isDialog]="true" [resourceId]="data.resourceId"></planet-resources-viewer>`
+  template: `
+  <div mat-dialog-content>
```

Comment:
The spacing and indentation is all off with this template.

1. The entire template should be 1 indentation in from the `template:` line
2. Every indentation step should only be 2 spaces, not 4.
3. Other than indentation there should only ever be 1 space in a row
4. In templates, there should be no spaces around an equals sign